v20.1/inverted-indexes.md

```diff
@@ -83,7 +87,10 @@ Tables are not locked during index creation thanks to CockroachDB's [schema chan
 Indexes create a trade-off: they greatly improve the speed of queries, but slightly slow down writes (because new values have to be copied and sorted). The first index you create has the largest impact, but additional indexes only introduce marginal overhead.
 
 ### Comparisons
-Currently, inverted indexes only support equality comparisons using the `=` operator. If you require comparisons using `>`, `<=`, etc., you can create an index on a computed column using your JSON payload, and then create a regular index on that. So if you wanted to write a query where the value of "foo" is greater than three, you would:
+
+#### JSONB
+
+Currently, inverted indexes on `JSONB` columns only support equality comparisons using the [`=` operator](functions-and-operators.html#operator-equals). If you require comparisons using [`<`](functions-and-operators.html#operator-less-than)), [`<=`](functions-and-operators.html#operator-less-than-or-equal), etc., you can create an index on a computed column using your JSON payload, and then create a regular index on that. So if you wanted to write a query where the value of "foo" is greater than three, you would:
```

Comment:
This is incorrect (and was before your changes) AFAICT. Only the @> variants (containment) are supported. You can't do a normal equality search using an inverted index at all.

Reply:
Ack!  Updated to say

> Inverted indexes on `JSONB` columns support the following comparison operators:
> - "is contained by": [`<@`](functions-and-operators.html#operator-is-contained-by)
> - "contains": [`@>`](functions-and-operators.html#operator-contains)